school-migration.sql

```diff
@@ -1,775 +0,0 @@
-DROP TABLE IF EXISTS school CASCADE;
```

Comment:
Why was the school-migration deleted? May be best to keep it for future but unsure on context here.

Reply:
This was my way of creating mock data
All these files should now be deleted and all the CREATE TABLE commands should be in 
server/db/migrations.sql

We don't want multiple CREATE TABLE commands in different places.
Keep them all in server/db/migrations.sql